imageviewer.js

```diff
@@ -740,49 +770,47 @@
         snapView: true,
         maxZoom: 500,
         refreshOnResize: true,
-        zoomOnMouseWheel : true
+        zoomOnMouseWheel: true
     }
 
-    window.ImageViewer = function (container, options) {
-        var imgElm, imgSrc, hiResImg;
-        if (!(container && (typeof container == "string" || container instanceof Element || container[0] instanceof Element))) {
-            options = container;
-            container = $('#iv-container');
+    window.ImageViewer = function(container, options) {
+        var imgElm, imgSrc, hiResImg
+        if (!(
+                container &&
+                (typeof container == 'string' || container instanceof Element || container[0] instanceof Element)
+            )) {
+            options = container
+            container = document.getElementById('iv-container')
         }
-
-        container = $(container);
-
-        if (container.is('img')) {
-            imgElm = container;
-            imgSrc = imgElm[0].src;
-            hiResImg = imgElm.attr('high-res-src') || imgElm.attr('data-high-res-src');
-            container = imgElm.wrap('<div class="iv-container" style="display:inline-block; overflow:hidden"></div>').parent();
-            imgElm.css({
-                opacity: 0,
-                position: 'relative',
-                zIndex: -1
-            });
+        if (typeof container === 'string') container = document.querySelector(container)
+
+        if (container.tagName === 'IMG') {
+            imgElm = container
+            imgSrc = imgElm.src
+            hiResImg = imgElm.getAttribute('high-res-src') || imgElm.getAttribute('data-high-res-src')
+            var parent = imgElm.parentNode
+            parent.removeChild(imgElm)
+            imgElm.style.opacity = 0
+            imgElm.style.position = 'relative'
+            imgElm.style.zIndex = -1
+
+            var div = document.createElement('div')
+            div.className = 'iv-container'
+            div.style.display = 'inline-block'
+            div.style.overflow = 'hidden'
+            div.appendChild(imgElm)
+            parent.appendChild(div)
+            container = div
         } else {
-            imgSrc = container.attr('src') || container.attr('data-src');
-            hiResImg = container.attr('high-res-src') || container.attr('data-high-res-src');
+            imgSrc = container.getAttribute('src') || container.getAttribute('data-src')
+            hiResImg = container.getAttribute('high-res-src') || container.getAttribute('data-high-res-src')
         }
 
+        var viewer = new ImageViewer(container, options)
+        viewer._init()
 
-        var viewer = new ImageViewer(container, options);
-        viewer._init();
-
-        if (imgSrc) viewer.load(imgSrc, hiResImg);
-
-        return viewer;
-    };
+        if (imgSrc) viewer.load(imgSrc, hiResImg)
 
-
-    $.fn.ImageViewer = function (options) {
```

Comment:
lets not remove this, we can check if jquery is available then add this. This will be helpful to make it backward compatible.